src/components/charts/BarChart.tsx

```diff
@@ -196,6 +196,10 @@ const BarChart = () => {
     );
     setIsLoading(false);
   };
+  useEffect(() => {
+    const removeEventListener = submitOnEnter(incrementCounter);
+    return removeEventListener;
+  } );
```

Comment:
why does this useEffect run on every render ?

Reply:
Added a dependency to it